max_lin_vel: 3.82
max_ang_vel: 3.82

lin_step: 0.1
ang_step: 0.1

cmd_vel_pub_rate: 10.0
cmd_vel_pub_timeout: 600.0
cmd_vel_pub_need_log_info: True
